app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -257,6 +257,11 @@ def combine_alert_data(all_alerts, alert_name)
             new_alert_data[:name] = value if key.end_with?('_AlertName')
             new_alert_data[:result] = value if key.end_with?('_AuthenticationResult')
             new_alert_data[:region] = value if key.end_with?('_Regions')
+            new_alert_data[:disposition] = value if key.end_with?('_Disposition')
```

Comment:
I tried running this locally and following the Testing Plan, but I don't see any `disposition` or `region` noted in my `log/events.log` file. I tried four types of failure `yml` file and one successful `yml` file (all from `/spec/fixtures`).

I'm guessing this is a `yml` file issue and not a code issue - but do you know what test file I can use to test this locally?

Reply:
@night-jellyfish, you cannot really test with local mock client, since it bypass both acuant or trueid client.

To see the event change you really need to use trueid client with mock service or in real deployment.

Reply:
That makes sense. Thank you for clarifying!